website/content/en/docs/building-operators/golang/references/env-test-setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: EnvTest Setup
+linkTitle: EnvTest Setup
+weight: 50
+---
+
+## Overview 
+
+This document describes how to configure the environment for the [controller tests][controller-test] supported by the SDK.
```

Comment:
Link to [envtest](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/envtest) somewhere in the overview.